sos/report/plugins/ceph.py

```diff
@@ -8,14 +8,15 @@
 
 from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
 from socket import gethostname
+import re
 
 
 class Ceph(Plugin, RedHatPlugin, UbuntuPlugin):
 
     short_desc = 'CEPH distributed storage'
 
     plugin_name = 'ceph'
-    profiles = ('storage', 'virt')
+    profiles = ('storage', 'virt', 'container')
```

Comment:
As an enablement trigger, you can use the `containers` class attr here. Also a tuple, elements should be patterns for container names.

Reply:
You would leave just 'container' as a profile?

Reply:
No, I'm saying you can do this:

```python
profiles = ('storage', 'virt', 'container')
containers = ('ceph-(mon|rgw|osd)*',)
```

So it will be enabled with the container profile, but _also_ if there are any running containers with names matching that pattern.

Reply:
OK clear now. Thx